packages/xml-lsp-server/test/server.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { expect } from "chai";
+
+import { validateDocument, SYNTAX_ERROR_MSG } from "../src/serverHelper";
```

Comment:
really really minor, but generally you want to first import
external deps and only then relative deps `../src/serverHelper`